.vscode/settings.json

```diff
@@ -0,0 +1,16 @@
+{
+    "cmake": {
+        "sourceDirectory": "${workspaceFolder}/src",
+        "buildDirectory": "${userHome}/geosx-build",
+        "cacheInit": "/workspaces/GEOSX/host-configs/environment.cmake",
+        "configureSettings": {
+            "BLT_MPI_COMMAND_APPEND": "--allow-run-as-root;--use-hwthread-cpus;--oversubscribe",
+            "ENABLE_OPENMP": "OFF",
+            "ENABLE_HYPRE": "OFF",
+            "ENABLE_TRILINOS": "ON",
```

Comment:
Are we sure we want to dev/try with `Trilinos` ? IIUC most users are using `HYPRE` isn't it @CusiniM  ?

Reply:
I don't know, we can change.

Reply:
@francoishamon What do you think is best?

Reply:
I agree with @jafranc, the default configuration should be hypre, this is the setting of most users

Reply:
done